transform.py

```diff
@@ -37,9 +39,13 @@
 
 # Catch any exception
 except Exception as e:
-    print(f"Error: {e}")
+    print("Error during transform")
     print("-" * 80)
     print("Full Log:")
     for entry in transform.error_log:
         print((f"{entry.filename}: {entry.line}, "
-              f"{entry.column}> {entry.message}>"))
+               f"{entry.column}> {entry.message}>"))
+    raise e
+
+ome_schema.validate(ome)
+print("OME XML valid, schema validation ok.")
```

Comment:
Nit: move this into the end of the `try`.

Reply:
I moved this out of the `try` intentionally. because the `except` has some special logic for parsing errors from the transform step. The errors thrown by `ome-types` if validation fails are already nicely formatted and don't need that.